templates/tools/dockerfile/interoptest/grpc_interop_go/Dockerfile.template

```diff
@@ -1,6 +1,6 @@
 %YAML 1.2
 --- |
-  # Copyright 2021 The gRPC Authors
+  # Copyright 2024 The gRPC Authors
```

Comment:
We shouldnt have to update the copyright everytime we modify the file.

Reply:
Reverted the change.